titan/params/partner_tracing.yml

```diff
@@ -1,39 +1,231 @@
 partner_tracing:
-  exposure:
-    description: "Which exposure is being partner traced"
-    type: enum
-    default: hiv
-    values:
-      - hiv
-  prob:
-    default: 1.0
-    description: Probability that an agent whose partner is diagnosed with hiv will become traced
-    type: float
-    min: 0.
-    max: 1.
-  dx_prob:
-    default: .947
-    description: Probability that an agent who is traced and has the exposure gets diagnosed with the exposure - default is  TP / (TP + FN)
-    type: float
-    min: 0
-    max: 1
-  bond_type:
-    description: "What type of partnerships are traced"
-    type: array
-    class: bond_types
-    default:
-      - Sex
-      - Inj
-      - SexInj
-  start_time:
-    description: "What time step to start partner tracing"
-    type: int
-    default: 0
-  stop_time:
-    description: "What time step to stop partner tracing"
-    type: int
-    default: 9999
-  trace_duration:
-    description: "For how many time steps after partner diagnosis a partner is traced"
-    type: int
-    default: 1
+    exposure:
+      description: "Which exposure is being partner traced"
+      type: enum
+      default: hiv
+      values:
+        - hiv
+    prob:
+      default: 1.0
+      description: Probability that an agent whose partner is diagnosed with hiv will become traced
+      type: float
+      min: 0.
+      max: 1.
+    dx_prob:
+      default: .947
+      description: Probability that an agent who is traced and has the exposure gets diagnosed with the exposure - default is  TP / (TP + FN)
+      type: float
+      min: 0
+      max: 1
+    bond_type:
+      description: "What type of partnerships are traced"
+      type: array
+      class: bond_types
+      default:
+        - Sex
+        - Inj
+        - SexInj
+    start_time:
+      description: "What time step to start partner tracing"
+      type: int
+      default: -12
+    stop_time:
+      description: "What time step to stop partner tracing"
+      type: int
+      default: 9999
+    trace_duration:
+      description: "For how many time steps after partner diagnosis a partner is traced"
+      type: int
+      default: 1
+      
+    white:
```

Comment:
this should be defined more similarly to the demographics, as a sub-dict with races as keys. As it is now, it's hardcoded to these specific races. See: https://github.com/pph-collective/TITAN/blob/main/titan/params/demographics.yml